test/Microsoft.AspNet.Mvc.IntegrationTests/BodyValidationIntegrationTests.cs

```diff
@@ -108,7 +108,7 @@ private class Address
         private class Person4
         {
             [FromBody]
-            [Required]
+            [BindingBehavior(BindingBehavior.Optional)]
```

Comment:
Changed an assortment of properties to have the behavior expected by integration tests (this change and all of the below).